andela/travel_tool_back
# #160710730 Add maintenance reason

#### What does this PR do?
Adds an endpoint to add the maintenance reason to the database.

#### Description of Task to be completed?
- Add an endpoint to add maintenance reason to the database
- Add tests for the new endpoint.

#### How should this be manually tested?
- Clone the repository and enter the project's folder
- Checkout to `ft-add-maintainance-reason-#160710730`
- Assign yourself the travel admin role by changing your roleId to 29187 in the users table
- Start the application using 'make start'
- Open postman and provide your token in the authorization header.
- If you do not have a guest house id, see [this](https://github.com/andela/travel_tool_back/pull/65) for guesthouse structure to create one
- Make a POST request to `/api/v1/room/:room-id/maintainance`. Ensure to add the `id` as the ID of any room.
`THE BODY STRUCTURE`
{
	"reason":"Broken windows",
	"start":"10/28/2018",
	"end":"2018-10-16"
}

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160710730](https://www.pivotaltracker.com/story/show/160710730)

#### Screenshots (if appropriate)
<img width="1236" alt="screen shot 2018-10-14 at 13 49 33" src="https://user-images.githubusercontent.com/23402614/46915725-0fc06800-cfb8-11e8-8739-e88c22fa8138.png">

#### Questions:
None
